benchmarks/dream-challenge/README.md

```diff
@@ -0,0 +1,19 @@
+### DREAM Somatic Mutation Calling Challenge Benchmarks
```

Comment:
Cool! When you have a chance (in this PR or another), could you add instructions for running a benchmark on google cloud?